terraform/shared/modules/newrelic/alerts.tf

```diff
@@ -0,0 +1,115 @@
+data "newrelic_entity" "gsa-fac" {
+  name   = "gsa-fac-${var.cf_space_name}"
+  type   = "APPLICATION"
+  domain = "APM"
+}
+
+resource "newrelic_alert_policy" "alert_policy" {
+  name = "${var.cf_space_name}-alert-policy"
+}
+
+resource "newrelic_notification_destination" "email_destination" {
+  account_id = var.new_relic_account_id
+  name       = "email_destination"
+  type       = "EMAIL"
+
+  property {
+    key   = "email"
+    value = "daniel.swick@gsa.gov, matthew.jadud@gsa.gov, alexander.steel@gsa.gov"
+  }
+}
+
+resource "newrelic_notification_channel" "email_channel" {
+  account_id     = var.new_relic_account_id
+  name           = "${var.cf_space_name}_email_notification_channel"
+  type           = "EMAIL"
+  product        = "IINT"
+  destination_id = newrelic_notification_destination.email_destination.id
+
+  property {
+    key   = "subject"
+    value = "{{issueTitle}}"
```

Comment:
Is this dynamic to the resource?

Reply:
yeah this inserts the title of the alert that triggered the notification as the email subject